console_logging.go

```diff
@@ -88,13 +88,24 @@ var (
 	Color = false
 )
 
+func IsValid(file *os.File) bool {
+	if file == nil {
+		return false
+	}
+	_, err := file.Stat()
+	return err == nil
```

Comment:
Don't you need to check more than that?

Such as the file is not a directory or a special file, or is readable

Reply:
This check is about not panicking/failing when os.StdErr doesn't exist (wasm case, or it's been .Close()ed) - it's to avoid the color mode in init failing